src/c_kzg_4844.c

```diff
@@ -1194,32 +1221,43 @@ C_KZG_RET compute_aggregate_kzg_proof(KZGProof *out,
     const Blob blobs[],
     size_t n,
     const KZGSettings *s) {
-  KZGCommitment* commitments = calloc(n, sizeof(KZGCommitment));
-  if (0 < n && commitments == NULL) return C_KZG_MALLOC;
+  C_KZG_RET ret;
+  Polynomial* polys = NULL;
+  KZGCommitment* commitments = NULL;
+
+  commitments = calloc(n, sizeof(KZGCommitment));
+  if (0 < n && commitments == NULL)
+  {
```

Comment:
nit: brace formatting inconsistent with the prevailing style

Reply:
Will fix. Also, I was planning to do a formatting PR later.

* Use consistent indentation (it's split between 4 spaces & 2 spaces).
* Use consistent brackets. For this PR I used the func's style.

Reply:
Will fix. Also, I plan to make a PR later to use consistent indentation. It's split between 4 spaces & 2 spaces.